aruna/api/internal/v1/proxy.proto

```diff
@@ -104,3 +116,32 @@ message CreatePresignedDownloadResponse {
 message CreateBucketRequest { string bucket_name = 1; }
 
 message CreateBucketResponse {}
+
+message DeleteObjectRequest {
+  Location location = 1;
+}
+
+message DeleteObjectResponse {}
+
+message MoveObjectRequest {
+  Location from = 1;
+  Location to = 2;
+}
+
+message MoveObjectResponse {}
+
+message FinalizeObjectRequest {
+  string object_id = 1;
```

Comment:
Is `object_id` necessary within this request?

Reply:
No this is not necessary, removed by [ed346f3](https://github.com/ArunaStorage/ArunaAPI/pull/58/commits/ed346f373029802d3f8fcbef7ab5044b56484d33)

Reply:
Actually, the whole purpose for this object_id was to indicate the temporary location where the object was initially stored in. So removing this id entirely will not work. 

I have changed this now to `before_location` and `after_location` to indicate the move in [5f4469a](https://github.com/ArunaStorage/ArunaAPI/pull/58/commits/5f4469a5e1aba5071f40d64019e3613783b64ee9). The before location will be determined by the backend, the final moved location by the data_proxy based on the SHA-256 hash.